superset-frontend/src/views/CRUD/alert/AlertList.tsx

```diff
@@ -397,7 +407,14 @@ function AlertList({
           },
         ]}
         buttons={subMenuButtons}
-      />
+      >
+        <RefreshContainer>
+          <LastUpdated
+            updatedAt={lastFetched || ''}
```

Comment:
I'm assuming you want a null value from moment, thus passing in the `''`?

Reply:
yup, in case lastFetched is still `null`.

Reply:
Actually we just add null to the Prop type, moment will treat `null` and `''` the same.